pkg/services/control/ir/types.proto

```diff
@@ -18,12 +18,15 @@ enum HealthStatus {
     // Undefined status, default value.
     HEALTH_STATUS_UNDEFINED = 0;
 
+    // Create IR server
+    CREATE_SERVER = 1;
```

Comment:
So it something in-between the IR's `New` and `Start`, right? What happens there? All of blockchain init?

Reply:
As I understand it, all initialization of blockchain, clients, sidechain, configurations and others takes place in `New` func of ir, so after this function it just goes `Strart`.